release_notes/gitops-release-notes-1-14.adoc

```diff
@@ -0,0 +1,36 @@
+//OpenShift GitOps Release Notes
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="gitops-release-notes"]
+= {gitops-title} release notes
+:context: gitops-release-notes
+
+toc::[]
+
+[NOTE]
+====
+For additional information about the OpenShift {gitops-shortname} life cycle and supported platforms, refer to the link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles] and link:https://access.redhat.com/support/policy/updates/openshift[Red{nbsp}Hat {OCP} Life Cycle Policy].
```

Comment:
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#lifecycle

the product names are probably fine 

```suggestion
For additional information about the OpenShift {gitops-shortname} lifecycle and supported platforms, refer to the link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles] and link:https://access.redhat.com/support/policy/updates/openshift[Red{nbsp}Hat {OCP} Life Cycle Policy].
```

Reply:
Fixed.